### ISSUE TITLE
[Code Addition Request]: Add Word Games with lives

### ISSUE THREAD

**Issue by souvikpramanikgit** [UTSAVS26/PyVerse#880]
### Have you completed your first issue?

- [X] I have completed my first issue

### Guidelines

- [X] I have read the guidelines
- [X] I have the link to my latest merged PR

### Latest Merged PR Link

https://github.com/UTSAVS26/PyVerse/pull/794#event-14796558650

### Project Description

This game is a guessing game where the player tries to figure out a secret word, one letter at a time. The twist? You have a limited number of guesses, represented by lives.

Basically, the objective is to guess the secret word before running out of lives.

![image](https://github.com/user-attachments/assets/93bd89cb-a721-48a0-823d-19444e8f2dd4)

### Full Name

Souvik Kumar Pramanik

### Participant Role

GSSOC
